src/mono/mono/eglib/glib.h

```diff
@@ -1004,7 +1004,8 @@ typedef enum {
 	G_CONVERT_ERROR_FAILED,
 	G_CONVERT_ERROR_PARTIAL_INPUT,
 	G_CONVERT_ERROR_BAD_URI,
-	G_CONVERT_ERROR_NOT_ABSOLUTE_PATH
+	G_CONVERT_ERROR_NOT_ABSOLUTE_PATH,
+	G_CONVERT_ERROR_ALLOC_FAILED
```

Comment:
gtk glib uses `G_CONVERT_ERROR_NO_MEMORY` - https://developer.gnome.org/glib/stable/glib-Character-Set-Conversion.html#GConvertError maybe we shoudl use the same name.